django_x509/base/models.py

```
@@ -213,57 +191,77 @@ def save(self, *args, **kwargs):
+        if self.certificate:
             try:
-                args = (crypto.FILETYPE_PEM, getattr(self, field))
-                kwargs = {}
-                if method_name == "load_privatekey":
-                    kwargs["passphrase"] = getattr(self, "passphrase").encode("utf8")
-                load_pem(*args, **kwargs)
-            except OpenSSL.crypto.Error as e:
-                error = "OpenSSL error: <br>{0}".format(
-                    str(e.args[0]).replace("), ", "), <br>").strip("[]")
+                x509.load_pem_x509_certificate(self.certificate.encode("utf-8"))
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)